Logger/BaseLoggerMixins.cs

```diff
@@ -1,7 +1,47 @@
-﻿namespace Logger
+﻿using System;
+
+namespace Logger
 {
     public static class BaseLoggerMixins
     {
-        
+        public static void Error(this BaseLogger t, string message, params object [] args)
+        {
+            if (t==null)
+            {
+                throw new ArgumentNullException(nameof(t));
+            }
+
+            t.Log(LogLevel.Error, String.Format("{0} {1}", message, LogLevel.Error));
```

Comment:
From the coding guidelines

> DO use the C# keyword rather than the BCL name when specifying a data type (e.g., string rather than String).

In addition you are dropping the `args` parameter.

```suggestion
            t.Log(LogLevel.Error, string.Format(message, args));
```

Reply:
understood.